b2sdk/bucket.py

```diff
@@ -750,6 +750,34 @@ def hide_file(self, file_name):
         response = self.api.session.hide_file(self.id_, file_name)
         return FileVersionInfoFactory.from_api_response(response)
 
+    def copy_file(
+        self,
+        file_id,
+        new_file_name,
+        bytes_range=None,
+        metadata_directive=None,
+        content_type=None,
+        file_info=None,
+    ):
+        """
+        Creates a new file by copying from an existing file.
+
+        :param str file_id: file ID of existing file
+        :param str new_file_name: file name of the new file
+        :param tuple[int, int], None bytes_range: start and end offsets
+        :param str, None metadata_directive: `'COPY'` or `'REPLACE'`, default is `'COPY'`
```

Comment:
Should be an enum. Make a # TODO comment to indicate that we will change that soon when the enum34 PR is merged.